srcpkgs/volta/template

```diff
@@ -1,26 +1,18 @@
 # Template file for 'volta'
 pkgname=volta
-version=0.9.3
+version=0.9.99
```

Comment:
All this done, according to https://github.com/volta-cli/volta/releases/tag/v0.9.99 , this is a pre-release version. I don't think we should be shipping it.

Reply:
I will leave that decision to you. I haven't found any regressions since yesterday morning. The new support for npm link is pretty useful in some cases, though, but I guess we can wait the few days until 1.0.0 is released.